vsutil.py

```diff
@@ -1,58 +1,63 @@
 """
-VSUtil. A collection of general-purpose Vapoursynth functions to be reused in modules and scripts.
+VSUtil. A collection of general-purpose VapourSynth functions to be reused in modules and scripts.
 """
+__all__ = ['core', 'fallback', 'frame2clip', 'get_depth', 'get_plane_size', 'get_subsampling', 'get_w', 'get_y',
+           'insert_clip', 'is_image', 'iterate', 'join', 'plane', 'split', 'vs']
+
 from functools import reduce
-from typing import Callable, TypeVar, Union, List, Tuple, Optional
+from mimetypes import types_map
+from os import path
+from typing import Callable, List, Optional, Tuple, TypeVar, Union
+
 import vapoursynth as vs
-import mimetypes
-import os
 
 core = vs.core
 T = TypeVar("T")
 
 
 def get_subsampling(clip: vs.VideoNode) -> str:
     """
-    Returns the subsampling of a clip in human-readable format.
+    Returns the subsampling of a VideoNode in human-readable format.
     """
+    if clip.format.color_family not in (vs.YUV, vs.YCOCG):
+        return ''
```

Comment:
I don't know about this. When would you be using this function on a clip without subsampling and would you expect an empty string as the return value? Since I don't know, I'm not blocking this, of course, but it does raise a question to me.

Reply:
I was thinking if some function has two different ways of handling clips with/without subsampling, they could `if get_subsampling(clip): do something...` instead raising an error. And figured we wouldn't want two return types for such a simple function (a `False` here for example).

I also am going to later suggest a `get_color_family(clip: vs.VideoNode) -> str: return clip.format.color_family.name` function for vsutil that this function could make use of and maybe some other scripts as well.

Reply:
I want to suggest returning `None` in this case, because … well, it doesn't have any subsampling in that case. If people want to have that render as '', they can `f"{get_subsampling(clip) || ''}"`, for example.

Reply:
Makes sense, but what should we reword `Returns the subsampling of a VideoNode in human-readable format.` to then? Or should we just leave it and just let the typehint `-> Union[None, str]` speak for itself.

Reply:
Typehint speaks for itself imo, but could add `Returns None for formats without subsampling.` regardless.

Reply:
Used your suggestion in b408157, ready for merge now I believe.

```
============================= test session starts ==============================
platform linux -- Python 3.8.2, pytest-5.4.1, py-1.8.1, pluggy-0.13.1 -- /usr/bin/python3.8
cachedir: .pytest_cache
collecting ... collected 11 items

test_vsutil.py::VsUtilTests::test_depth PASSED                           [  9%]
test_vsutil.py::VsUtilTests::test_fallback PASSED                        [ 18%]
test_vsutil.py::VsUtilTests::test_frame2clip PASSED                      [ 27%]
test_vsutil.py::VsUtilTests::test_get_w PASSED                           [ 36%]
test_vsutil.py::VsUtilTests::test_get_y PASSED                           [ 45%]
test_vsutil.py::VsUtilTests::test_insert_clip PASSED                     [ 54%]
test_vsutil.py::VsUtilTests::test_is_image PASSED                        [ 63%]
test_vsutil.py::VsUtilTests::test_iterate PASSED                         [ 72%]
test_vsutil.py::VsUtilTests::test_plane_size PASSED                      [ 81%]
test_vsutil.py::VsUtilTests::test_split_join PASSED                      [ 90%]
test_vsutil.py::VsUtilTests::test_subsampling PASSED                     [100%]

============================== 11 passed in 0.10s ==============================
```